client/src/components/modal-forms/ArchiveHiredParticipantForm.js

```diff
@@ -27,78 +59,110 @@ const archiveHiredParticipantInitialValues = {
   confirmed: false,
 };
 
-export const ArchiveHiredParticipantForm = ({ onSubmit, onClose }) => {
+export const ArchiveHiredParticipantForm = ({ onSubmit, onClose, participantId }) => {
+  const [typeOptions, setTypeOptions] = useState([]);
+  const [endDate, setEndDate] = useState([]);
+
+  useEffect(() => {
+    fetchFormOptionData(participantId).then(({ typeOptions, endDate }) => {
+      setTypeOptions(typeOptions);
+      setEndDate(endDate);
+    });
+  }, [setTypeOptions, setEndDate, participantId]);
+
+  const getStatusOptions = (selectedReason) => {
+    if (typeOptions.includes(ROSCompletedType)) {
+      return selectedReason === ROSReason
+        ? formatOptions([ROSCompleteStatus])
+        : formatOptions([ROSUnderwayStatus]);
+    } else {
```

Comment:
We don't need `else` with early return

Reply:
nice catch @alva-fresh

Reply:
Whoops, yeah, that's leftover from when I had a different structure here! Thanks, I'll get that fixed!